api/lib/redisHelper.mjs

```diff
@@ -1,8 +1,9 @@
 import config from 'config';
 import dotenv from 'dotenv';
 import MisformedKeyError from '../errors/redis/MisformedKeyError.js';
-
+import StudentNotFoundError from '../lib/HttpErrors/StudentNotFoundError.js';
```

Comment:
Redis should not be throwing HTTP errors (see how other errors are handled for examples).